canonical/snapcraft.io
# Add signing key endpoints

## Done
- Add the following endpoints:
    - [GET] /admin/store/<store_id>/signing-keys
    - [POST] /admin/store/<store_id>/signing-keys
    - [DELETE] /admin/store/<store_id>/signing-keys/<signing-key-sha3-384>
## How to QA
- GET: go to https://snapcraft-io-4338.demos.haus/admin/store/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/signing-keys
- POST: should be tested from a frontend client with payload {"name": string}
- DELETE: Should also be tested from a frontend client
## Issue / Card https://warthogs.atlassian.net/browse/WD-4660
Fixes #

## Screenshots
